demos/rag_agentic/notebooks/Level5_agentic_and_mcp.ipynb

```diff
@@ -5,23 +5,24 @@
    "id": "b5ecd807-fb1c-41eb-a948-365e57396d90",
    "metadata": {},
    "source": [
-    "# Level 5: Agentic & MCP (Medium Difficulty)\n",
+    "# Level 5: Agents & MCP Tools\n",
     "\n",
-    "This tutorial is for developers who are already familiar with [basic Agentic workflows](./Level2_simple_agentic_with_websearch.ipynb). \n",
-    "This tutorial will highlight a couple of slightly more advanced use cases for agents where a single tool call is insufficient to complete the required task. Here we will rely on both sequential agent calls and conditional logic to expand our agents capabilities. \n",
+    "This notebook is for developers who are already familiar with [basic Agentic workflows](Level2_simple_agentic_with_websearch.ipynb). \n",
+    "Here, we will highlight more advanced use cases for agents where a single tool call is insufficient to complete the required task.\n",
     "\n",
-    "We will also use [MCP tools](https://github.com/modelcontextprotocol/servers) hosted locally or on an OpenShift cluster throughout this demo to showcase how users can go beyond Llama Stacks's current set of builtin tools to connect to many different services and data sources to build their own custom agents.  \n",
+    "We will also use [MCP tools](https://github.com/modelcontextprotocol/servers) (which can be deployed onto an OpenShift cluster) throughout this demo to show users how to extend their agents beyond Llama Stacks's current builtin tools and connect to many different services and data sources to build their own custom agents.  \n",
     "\n",
     "### Agent Example:\n",
     "\n",
-    "This notebook will walkthrough how to build a system that can answer the following question via agents built with Llama Stack:\n",
+    "This notebook will walkthrough how to build a system that can answer the following question via an agent built with Llama Stack:\n",
     "\n",
-    "- [*\"Review OpenShift logs for pods node-123 and node-456. Categorize each as ‘Normal’ or ‘Error’. If any show ‘Error’, send a Slack message to the ops team. Otherwise, show a simple summary.\"*](#summarize-any-errors-that-might-be-happening-on-my-openshift-cluster)\n",
+    "- [*\"Review OpenShift logs for pod-123 and pod-456. Categorize each as either ‘Normal’ or ‘Error’. If any are considered to be ‘Error’, send a Slack message to the ops team. Otherwise, show a simple summary.\"*](#summarize-any-errors-that-might-be-happening-on-my-openshift-cluster)\n",
```

Comment:
Noticed that the example question is hyperlinked to a markdown anchor (#summarize-any-errors-that-might-be-happening-on-my-openshift-cluster), but that anchor does not seem to exist in this notebook.
When clicked, it just jumps to notebook dir. Should we remove the hyperlink, since its just an illustrative question and not meant to link anywhere?